packages/editor/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@portabletext/editor",
-  "version": "1.2.0",
+  "version": "1.1.12-canary.3",
```

Comment:
Do we want to merge with this change? I don't understand...

Reply:
Release please will correct it later so it's fine :)